src/extension.ts

```diff
@@ -65,11 +68,22 @@ function activateTOC(
   // Add the ToC widget to the application restorer:
   restorer.add(toc, 'juputerlab-toc');
 
+  // Attempt to load plugin settings:
+  let settings: ISettingRegistry.ISettings | undefined;
```

Comment:
```suggestion
  let settings: ISettingRegistry.ISettings;
```

Reply:
I had to add `| undefined` because it was causing a compiler error.

Reply:
That is odd. We should not need to explicitly type as `undefined`. What was the compile error?

Reply:
```
Variable 'settings' is used before being assigned.

let settings: ISettingRegistry.ISettings
```

Reply:
Okay. We'd probably need to modify a compiler setting to get this to go away. Should be fine to leave as is, for now.